src/workflows-app/SubmissionHistory.js

```diff
@@ -237,10 +238,21 @@ export const BaseSubmissionHistory = ({ namespace, workspace }, _ref) => {
                                       {
                                         style: { fontSize: 15 },
                                         disabled:
+                                          !permissionToAbort ||
                                           isRunSetInTerminalState(paginatedPreviousRunSets[rowIndex].state) ||
                                           paginatedPreviousRunSets[rowIndex].state === 'CANCELING',
-                                        tooltip:
-                                          isRunSetInTerminalState(paginatedPreviousRunSets[rowIndex].state) && 'Cannot abort a terminal submission',
+                                        tooltip: Utils.cond(
+                                          [
+                                            isRunSetInTerminalState(paginatedPreviousRunSets[rowIndex].state),
+                                            () => 'Cannot abort a terminal submission',
+                                          ],
+                                          [
+                                            paginatedPreviousRunSets[rowIndex].state === 'CANCELING',
+                                            () => 'Cancel already requested on this submission.',
+                                          ],
+                                          [!permissionToAbort, () => 'You must be the original submitter to abort this submission'],
```

Comment:
I guess it could have been. I still lean towards this because it's more generally correct / going to be correct in the future. (And if a non-creator starts a workflow, the workspace creator won't have the ability to stop it, so saying that here feels a little bit less true)